systemtest/src/main/java/io/strimzi/systemtest/utils/ClientUtils.java

```diff
@@ -43,15 +45,17 @@ public static void waitUntilClientReceivedMessagesTls(KafkaClientOperations kafk
     }
 
     public static void waitTillContinuousClientsFinish(String producerName, String consumerName, String namespace, int messageCount) {
-        long timeout = (long) messageCount * 1000;
+        // need to add at least 1-2minutes for finishing the jobs
+        long timeout = (long) messageCount * 1000 + Duration.ofMinutes(2).toMillis();
```

Comment:
any reason why you change this again? I thought we agreed it's not needed in this method.

Reply:
well I changed this as I'm using it in the test. I didn't wanted to add something like `messagesCount * 3000`, because this can cause unnecessarily long timeout in upgrade tests. So I added just two minutes to both function -> for finishing the job. If you agree I can leave it like this. If not, I'll change it

Reply:
Not great, not terrible. Maybe we should have for this some method which can be called in both cases?

Reply:
sure good idea